packages/kcms/src/match/service/generateRanking.ts

```diff
@@ -92,9 +92,9 @@ export class GenerateRankingService {
       const prevGoalTime = rankingData[i - 1]?.goalTimeSeconds ?? Infinity;
       const rank = rankingData[i - 1]?.rank ?? 0;
```

Comment:
```suggestion
      const prevRank = rankingData[i - 1]?.rank ?? 0;
```
`currentRank`と命名を揃えたほうがいい感じ